docs/sources/setup-grafana/configure-grafana/_index.md

```diff
@@ -964,6 +964,10 @@ This setting is ignored if multiple OAuth providers are configured. Default is `
 How many seconds the OAuth state cookie lives before being deleted. Default is `600` (seconds)
 Administrators can increase this if they experience OAuth login state mismatch errors.
 
+### oauth_login_error_message
+
+A custom error message for when users are unauthorized. Default is a key for an internationalized phrase in the frontend, `Login provider denied login request`.
```

Comment:
Is this just a technicality? Technically the default is "oauth.login.error", which is then set as `Login provider denied login request` or a translated version.

Reply:
This is fine imho, the internal string isn't something an end user really needs to know or care about.